hedera-node/src/test/java/com/hedera/services/state/merkle/MerkleNetworkContextTest.java

```diff
@@ -96,8 +99,7 @@ void setup() {
 				Instant.ofEpochSecond(1_234_789L, 12345)
 		};
 
-		consensusTimeOfLastHandledTxn = RichInstant.fromJava(
-				Instant.ofEpochSecond(1_234_567L, 54321));
+		consensusTimeOfLastHandledTxn = Instant.ofEpochSecond(1_234_567L, 54321);
```

Comment:
and some other long numbers like this 54321L in this test.

Reply:
Done.